bridetalk_project/src/components/molecule/forms/createuserform/CreateUserForm.jsx

```diff
@@ -1,70 +1,113 @@
-import {
-  Headline1,
-  EnterText,
-  EnterEmail,
-  EnterPassword,
-  DropdownLocation,
-  PrimaryButton,
-  CancelButton,
-} from "../../../atoms";
-
+import { Link, useNavigate } from "react-router-dom";
+import { useState } from "react";
+import { InputField, DropdownLocation, Button } from "../../../atoms";
 import "./createuserform.css";
-import { Link } from "react-router-dom";
 
-import { useNavigate } from "react-router-dom";
+import Parse from "parse";
+
 export const CreateUserForm = () => {
   let navigate = useNavigate();
 
-  const toFrontPage = () => {
-    let path = "/";
+  const navigateTo = (path) => {
     navigate(path);
   };
 
-  const toLoginPage = () => {
-    let path = "/login";
-    navigate(path);
-  };
+  const [username, setUsername] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPasword] = useState("");
+  const [location, setLocation] = useState("");
 
-  const goBack = () => {
-    let path = navigate(-1);
-    navigate(path);
+  //Todo: If user does not input a valid e-mail the user should not be created/save, and a alert should be send.
+
+  const addUser = async function () {
+    const userNameValue = username;
+    const passWordValue = password;
+
+    try {
+      // create a new Parse User instance and since the signUp method returns a Promise, we need to call it using await
+
+      const createdUser = await Parse.User.signUp(userNameValue, passWordValue);
```

Comment:
Nice. And nice with the comment above.
If one of the collaborators doesn't know why you use async, this will give them the understanding that they need.